chrome_extension/background/event.js

```diff
@@ -124,7 +116,7 @@ mooltipassEvent.onSaveSettings = function(callback, tab, settings) {
 	mooltipassEvent.onLoadSettings();
 }
 
-mooltipassEvent.onGetStatus = function(callback, tab) {
+mooltipassEvent.onGetStatus = function(callback, senderTab, tab) {
```

Comment:
senderTab is the popup. We do not need this in the function.
Args are passed after callback and senderTab, so the variable "tab" points to activeTab from the args